actors/common/src/main/java/org/sunbird/learner/actors/coursebatch/CourseBatchNotificationActor.java

```diff
@@ -43,6 +44,11 @@
   private DecryptionService decryptionService =
       org.sunbird.common.models.util.datasecurity.impl.ServiceFactory.getDecryptionServiceInstance(
           null);
+  private static String courseBatchNotificationSignature =
```

Comment:
Why JsonKey.SUNBIRD_COURSE_BATCH_NOTIFICATION_SIGNATURE is required? How are the existing email template signatures working without this? Where is the default value defined for this variable?

Reply:
Existing email template are using ORG name . Product Manager want it configurable. 
Default value is sunbird, it is added to env variable tracking google doc. and also informed in devops channel

Reply:
Is this configurable per tenant?

Reply:
Yes